leadership.html

```diff
@@ -250,79 +251,72 @@ <h1>رهبری</h1>
+                <br />از دیگران حمایت کرده و آن‌ها را پشتیبانی می‌کند در جهت رشد آن‌ها و درک بهتر یک موضوع فنی برای آن ها ابزارهایی فراهم می‌کند.
               </li>
             </ul>
           </div>
 
           <div class="team">
-            <span class="pill">Team Leading</span>
+            <span class="pill">رهبری تیم</span>
             <ul>
               <li>
-                <span class="highlight underline">Motivating Others &amp; Driving Alignment</span>
-                Understands and works towards the <span class="highlight">strategy and core principles</span> that align
-                to company goals.
+                <span class="highlight underline">انگیزه دادن به دیگران و هماهنگ‌سازی</span>
+                <span class="highlight">ِراهبردها و مبانی اصلی</span> هم‌راستا با اهداف شرکت را درک کرده و در در راستای آن‌ها کار می‌کند،
               </li>
               <li>
-                <span class="highlight underline">Expectation Management</span>
-                Takes responsibility for delivering on time, proactively communicates progress of their
-                work and issues that arise.
+                <span class="highlight underline">مدیریت انتظارات</span>
+                مسئولیت تحویل به موقع را بر عهده می گیرد و فعالانه پیشرفت کار و مسائلی که پیش می‌آید را اطلاع‌رسانی می‌کند.
               </li>
             </ul>
           </div>
-          <a class="link" id="show-senior-all" onclick="showAll('senior')">+ Show All Competencies</a>
-          <a class="link hide" id="hide-senior-all" onclick="hideAll('senior')">+ Hide Other Competencies</a>
+          <a class="link" id="show-senior-all" onclick="showAll('senior')">+ نمایش تمام شایستگی‌ها</a>
+          <a class="link hide" id="hide-senior-all" onclick="hideAll('senior')">+ پنهان کردن شایستگی‌های دیگر</a>
         </div>
         <div class="right col">
-          <div class=title>Examples</div>
-          <a class="show link" id="senior-show" onclick="showExamples('senior')">+ show examples</a>
-          <a class="hide link" id="senior-hide" onclick="hideExamples('senior')">- hide examples</a>
+          <div class=title>مثال‌ها</div>
+          <a class="show link" id="senior-show" onclick="showExamples('senior')">+ نمایش مثال‌ها</a>
+          <a class="hide link" id="senior-hide" onclick="hideExamples('senior')">- پنهان کردن مثال‌ها</a>
           <ul class="hide" id="senior-examples">
-            <li>Answers questions on a technical topic and provides useful resources for others to learn</li>
+            <li>پاسخ به سوالات مربوط به یک موضع فنی و فراهم کردن منابع مفید برای یادگیری دیگران</li>
             <li>Takes ownership of a converting from Flow to Typescript and helps others with the migration</li>
-            <li>Runs a guild meeting</li>
-            <li>Pairs with a team member to teach them what is and how to use Kubernetes</li>
+            <li>برگزاری جلساتِ صنفیRuns a guild meeting</li>
```

Comment:
I'm not sure about this translation, need your opinion to provide a better sentence

Reply:
I'm not sure either, but it can probably be translated like this:
با یک عضو تیم همراه می‌شود تا به آن یاد بدهد که کوبرنیز چی هست و چطور میشود از آن استفاده کرد.